react-js-gmp/src/services/movieService.js

```diff
@@ -1,17 +1,19 @@
+let moviesList = [];
```

Comment:
lets hardcode movies for now here and do not use this fetchMovies, more ower i cant check how it works because you didn't commited your api